src/commands/lambda/prompt.ts

```diff
@@ -189,6 +213,26 @@ export const requestDatadogEnvVars = async () => {
   }
 }
 
+export const requestEnvServiceVersion = async () => {
+  try {
+    const envQuestionAnswer = await prompt(envQuestion)
+    const inputedEnvQuestionAnswer = envQuestionAnswer[ENVIRONMENT_ENV_VAR]
+    process.env[ENVIRONMENT_ENV_VAR] = inputedEnvQuestionAnswer
+
+    const serviceQuestionAnswer = await prompt(serviceQuestion)
+    const inputedServiceQuestionAnswer = serviceQuestionAnswer[SERVICE_ENV_VAR]
+    process.env[SERVICE_ENV_VAR] = inputedServiceQuestionAnswer
+
+    const versionQuestionAnswer = await prompt(versionQuestion)
+    const inputedVersionQuestionAnswer = versionQuestionAnswer[VERSION_ENV_VAR]
+    process.env[VERSION_ENV_VAR] = inputedVersionQuestionAnswer
+  } catch (e) {
+    if (e instanceof Error) {
```

Comment:
Why that check?

Reply:
honestly I just copied the convention from the other try catch statements such as  https://github.com/DataDog/datadog-ci/blob/4b50c3b591ffcef1d3aca4876d6dc2e813b5bcf2/src/commands/lambda/prompt.ts#L181